src/main/java/org/sagebionetworks/bridge/spring/controllers/PermissionController.java

```diff
@@ -0,0 +1,87 @@
+package org.sagebionetworks.bridge.spring.controllers;
+
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.models.permissions.Permission;
+import org.sagebionetworks.bridge.services.PermissionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+import static org.sagebionetworks.bridge.Roles.ADMIN;
+
+@CrossOrigin
+@RestController
+public class PermissionController extends BaseController{
+    
+    private PermissionService permissionService;
+    
+    @Autowired
+    final void setPermissionService(PermissionService permissionService) {
+        this.permissionService = permissionService;
+    }
+    
+    @GetMapping("v1/permissions/{userId}")
+    public Set<Permission> getPermissionsForUser(@PathVariable String userId) {
+        UserSession session = getAuthenticatedSession(ADMIN);
+        
+        String appId = session.getAppId();
+        
+        return permissionService.getPermissionsForUser(appId, userId);
+    }
+    
+    @GetMapping("v1/permissions/{permissionType}/{objectId}")
+    public Set<Permission> getPermissionsForObject(
+            @PathVariable String permissionType, @PathVariable String objectId) {
+        UserSession session = getAuthenticatedSession(ADMIN);
+        
+        String appId = session.getAppId();
+        
+        return permissionService.getPermissionsForObject(appId, permissionType, objectId);
+    }
```

Comment:
Need to make a PermissionInfo object so that an account ref can be included in these responses.